src/app/app.component.html

```diff
@@ -21,22 +32,14 @@ <h2>
     {{ items.length }} <span *ngIf="items.length === 1; else elseBlock"> item </span>
     <ng-template #elseBlock>Items</ng-template>
   </h2>
-  <ul>
+  <div class="card" *ngIf="items.length > 0">
```

Comment:
Consider replacing `items.length > 0` with `items.length`. This way, `items.length` evaluates to a boolean value of `false` only if there is no item in the list. Otherwise, it evaluates to `true`.
```suggestion
  <div class="card" *ngIf="items.length">
```